src/ssort/_requirements.py

```diff
@@ -32,6 +33,13 @@ def get_requirements(node: ast.AST) -> Iterable[Requirement]:
         yield from get_requirements(child)
 
 
+def _get_requirements_from_nodes(
```

Comment:
I've hopefully made the code a little more simple with a _get_requirements_from_nodes function.